client/cli/go/cmds/heketi_storage.go

```diff
@@ -246,6 +246,9 @@ func createHeketiCopyJob(volume *api.VolumeInfoResponse) *batch.Job {
 	job.Spec.Parallelism = &p
 	job.Spec.Completions = &c
 	job.Spec.Template.ObjectMeta.Name = HeketiStorageJobName
+	job.Spec.Template.Spec.NodeSelector = map[string]string{
+		"beta.kubernetes.io/os": "linux",
```

Comment:
Instead of hard-coding a particular selector, I think the better approach would be to add a new command line argument that allows specifying one or more selector and then this can be passed into the Spec like it is here.
Then for your use case you can specify linux os but another user could specify some other custom selector they'd find useful.